pkg/utils/apiutil/apiutil.go

```diff
@@ -65,6 +65,8 @@ const (
 	XRealIPHeader = "X-Real-Ip"
 	// XCallerIDHeader is used to mark the caller ID.
 	XCallerIDHeader = "X-Caller-ID"
+	// XPDHandleHeader is used to mark whether this request is handled by the PD.
+	XPDHandleHeader = "X-PD-Handle-By"
```

Comment:
Only for tests?

Reply:
Yes. For now, it's only being used for testing. If similar issues occur with other interfaces in the future, we'll be able to identify which PD returned the response.